docs/vdc/networking-virtual-datacenter.md

```diff
@@ -10,360 +10,375 @@ ms.date: 09/24/2018
+An [Azure site-to-site VPN][VPN] is an interconnection service over the internet between on-premises networks and the VDC. It's established through secure encrypted connections, IPsec/IKE tunnels. An Azure site-to-site connection is flexible and quick to create. This VPN doesn't require any further procurement as all connections connect over the internet.
 
-For large numbers of VPN connections [**Azure Virtual WAN**][vWAN] is a networking service that provides optimized and automated branch-to-branch connectivity through Azure. Virtual WAN lets you connect and configure branch devices to communicate with Azure. This can be done either manually, or by using preferred provider devices through a Virtual WAN partner. Using preferred provider devices allows you ease of use, simplification of connectivity, and configuration management. The Azure WAN built-in dashboard provides instant troubleshooting insights that can help save you time, and gives you an easy way to view large-scale Site-to-Site connectivity.
+For large numbers of VPN connections, [Azure Virtual WAN][vWAN] is a networking service that provides optimized and automated branch-to-branch connectivity through Azure. With Virtual WAN, you can connect and configure branch devices to communicate with Azure. This connection can be done either manually or by using preferred provider devices through a Virtual WAN partner. Preferred provider devices give you ease of use, simplification of connectivity, and configuration management. The Azure WAN built-in dashboard provides instant troubleshooting insights that can save you time. And the dashboard gives you an easy way to view large-scale site-to-site connectivity.
 
-[**ExpressRoute**][ExR] is an Azure connectivity service that lets you create private connections between VDC and the on-premises networks. ExpressRoute connections do not go over the public Internet, and offer higher security, reliability, and higher speeds (up to 10 Gbps) along with consistent latency. ExpressRoute is very useful for VDCs, as ExpressRoute customers can get the benefits of compliance rules associated with private connections. With [ExpressRoute Direct][ExRD] you can connect directly to Microsoft routers at 100Gbps for customer with larger bandwidth needs.
+With [ExpressRoute][ExR], an Azure connectivity service, you can create private connections between the VDC and on-premises networks. ExpressRoute connections don't go over the public internet. They offer higher security, reliability, and higher speeds, up to 10 Gbps, along with consistent latency. ExpressRoute is useful for VDCs as ExpressRoute customers get the benefits of compliance rules associated with private connections. [ExpressRoute Direct][ExRD] connects you directly to Microsoft routers at 100 Gbps for customers with larger bandwidth needs.
```

Comment:
"With [ExpressRoute][ExR], an Azure connectivity service, you can create private connections between the VDC and on-premises networks" should be "[ExpressRoute][ExR], an Azure connectivity service, enables private connections between the Enterprise's on-premises network and their VDC implementation."